### PR TITLE
portal 1.2.2 (new formula)

### PR DESCRIPTION
**Pull request by ZinoKader**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

[Portal](https://github.com/SpatiumPortae/portal) is a quick and easy command-line file transfer utility from any computer to another. More details in the repo.

### CODE REVIEW

File: Formula/portal.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-s -X main.version=v#{version}"
+    system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/portal/"
+  end
+
+  test do
+    # Simple version check test.
+    system "#{bin}/portal", "version"
```
Comment (SMillerDev) [inline Formula/portal.rb:23]: We should check the output then
```suggestion
    # Simple version check test.
    assert_match version.to_s shell_output("#{bin}/portal version")
```

File: Formula/portal.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    # Start a local relay server on an open port.
+    port=free_port
+    fork do
+      $stdout.reopen("/dev/null")
+      $stderr.reopen("/dev/null")
```
Comment (SMillerDev) [inline Formula/portal.rb:29]: Why is this needed?

File: Formula/portal.rb (hunk 1 above)
Comment (ZinoKader) [reply Formula/portal.rb]: Thank you! Fixed in [53f50ed](https://github.com/Homebrew/homebrew-core/pull/125727/commits/53f50ed2a56e63d3a9f9d31e405005d3040b6b90)

File: Formula/portal.rb [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  homepage "https://github.com/SpatiumPortae/portal"
+  url "https://github.com/SpatiumPortae/portal/archive/v1.2.2.tar.gz"
+  sha256 "94c334285925175edfc470239ea2cc73a0039f9487ce92e60764f0965aba66d4"
+  license "MIT"
+  head "https://github.com/SpatiumPortae/portal.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```
Comment (carlocab) [inline Formula/portal.rb:12]: Is this needed? Does `brew livecheck portal` return the wrong output without it?

Comment (ZinoKader) [reply Formula/portal.rb]: It was not needed, the default strategy seems to work.

**Review by carlocab — APPROVED** [review]
